lib/src/visitor/async_evaluate.dart

```diff
@@ -282,14 +427,52 @@ class _EvaluateVisitor
         callback);
   }
 
+  /// Loads the module at [url] and passes it to [callback].
+  ///
+  /// The [stackFrame] and [nodeForSpan] are used for the name and location of
+  /// the stack frame in which the new module is executed.
```

Comment:
Maybe clarify that the stack frame is why this passes `Module` to a callback instead of just returning `Future<Module>`? This confused me before I realized that